experimental/casemapping/src/casemapping.rs

```diff
@@ -154,27 +258,21 @@ impl CaseMapping {
     /// Returns the full uppercase mapping of the given string as a [`Writeable`].
     /// This function is context and locale sensitive.
     ///
-    /// See [`Self::to_full_uppercase_string()`] for the equivalent convenience function that returns a String
+    /// See [`Self::to_full_uppercase_string()`] for the equivalent convenience function that returns a String,
+    /// as well as for an example.
     pub fn to_full_uppercase<'a>(&'a self, src: &'a str) -> impl Writeable + 'a {
         self.data
             .get()
             .full_helper_writeable(src, self.locale, MappingKind::Upper)
     }
 
-    /// Returns the full titlecase mapping of the given string as a [`Writeable`].
-    /// This function is context and locale sensitive.
-    ///
-    /// See [`Self::to_full_titlecase_string()`] for the equivalent convenience function that returns a String
-    pub fn to_full_titlecase<'a>(&'a self, src: &'a str) -> impl Writeable + 'a {
-        self.data
-            .get()
-            .full_helper_writeable(src, self.locale, MappingKind::Title)
-    }
-
     /// Case-folds the characters in the given string as a [`Writeable`].
     /// This function is locale-independent and context-insensitive.
     ///
-    /// See [`Self::full_fold_string()`] for the equivalent convenience function that returns a String
+    /// Can be used to test if two strings are case-insensitively equivalent.
```

Comment:
```suggestion
    /// Can be used to test if two strings are case-insensitively equivalent.
    /// Specifically, this function implements the operation [toCasefold](https://www.unicode.org/versions/Unicode15.0.0/ch03.pdf#G53253), and can be
    // used to implement [Default Caseless Matching](https://www.unicode.org/versions/latest/ch03.pdf#G34145).
```
It is important to be specific here, because there is also canonical caseless match, compatibility caseless match, and identifier caseless match, all of which are some flavour of case-insensitive equivalence.